packages/muelu/src/Graph/UncoupledAggregation/MueLu_UncoupledAggregationFactory_kokkos_def.hpp

```diff
@@ -234,14 +235,56 @@ namespace MueLu {
       Kokkos::deep_copy(aggStat, aggStatHost);
     }
 
-
     const RCP<const Teuchos::Comm<int> > comm = graph->GetComm();
     GO numGlobalRows = 0;
     if (IsPrint(Statistics1))
       MueLu_sumAll(comm, as<GO>(numRows), numGlobalRows);
 
+    LO numNonAggregatedNodes = numRows;
+    std::string aggAlgo = pL.get<std::string>("aggregation: coloring algorithm");
```

Comment:
@brian-kelley Thank you for this enhancement!  I don't see this option in muelu/doc/masterlist.xml, which it would need to be in order to parsed from MueLu's ParameterListInterpreter.    If you add the option to `masterlist.xml, then run `update_params.sh` from that directory, it will generate documentation and add it to the parser.